Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -9,7 +9,7 @@ The Mixed Reality Toolkit (MRTK) is a cross-platform toolkit for building Mixed
 To get started with the Mixed Reality Toolkit you will need:
 
 * [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
-* [Unity 2018.4.x](https://unity3d.com/get-unity/download/archive)
+* [Unity 2018.4.x](https://unity3d.com/get-unity/download/archive) or [Unity 2019.x](https://unity3d.com/get-unity/download/archive)
```

Comment:
It would be helpful to be specific about the versions (i.e. 2019.2 and 2019.1, but not 2019.3).

Reply:
will update. thx!